### ISSUE TITLE
Bot erroneously closing issues

### ISSUE THREAD

**Issue by lma-gh** [bromite/bromite#1763]
### Bromite version

Version: `89.0...` 
Arch: `arm` or `arm64` or `x86` 
Android version: `9.0` 
Device model: n/a 

### Flags

These are the flags changed under `chrome://flags`:

### Is this bug about the SystemWebView?

No

### Is the bug reproducible with latest version?

Yes

### Can the bug be reproduced with corresponding Chromium version?

Yes

### Is this bug happening in an incognito tab?
Yes

### Allow ads for the site; is the bug caused by the adblocker?
No

### Is the bug a crash?

No

### Describe the bug

The bot automatically closes issues, if it *thinks* that the issue does not contain the issue-template, even if it is present.
.
See #1762.
.
TL;DR: Please re-open #1760.

### Steps to reproduce the bug

Steps to reproduce the bug:
1. Open issue, including the issue-template.
2. Watch it being tagged as "missing issue-template" and being closed by the bot.

### Expected behavior

The bot should not tag and close an issue, if the issue does in fact contain the issue-template.

**Comment by kokukyocho** [comment]
Hello @lma-gh!

I am a bot :robot:

Thanks for submitting this issue!
I noticed that it is missing the template, please edit the issue to match the template for either a [bug](https://github.com/bromite/bromite/blob/master/.github/ISSUE_TEMPLATE/bug_report.md) or a [feature request](https://github.com/bromite/bromite/blob/master/.github/ISSUE_TEMPLATE/feature_request.md); the issue will be automatically re-opened afterwards.

If this is not an issue nor a feature request and you want to start a discussion then you can use [GitHub Discussions](https://github.com/bromite/bromite/discussions).

**Tip**: if you use GitHub in Desktop mode the template will be proposed to you when submitting issues.
